strr-web/components/bcros/form-section/property/Details.vue

```diff
@@ -104,6 +159,23 @@ const ownershipType = defineModel<string>('ownershipType')
 const businessLicense = defineModel<string>('businessLicense')
 const parcelIdentifier = defineModel<string>('parcelIdentifier')
 const businessLicenseExpiryDate = defineModel<string>('businessLicenseExpiryDate')
+const rentalUnitSpaceType = defineModel<string>('rentalUnitSpaceType')
+const isUnitOnPrincipalResidenceProperty = defineModel<boolean>('isUnitOnPrincipalResidenceProperty')
+const hostResidence = defineModel<string | null>('hostResidence')
+
+if (!formState.propertyDetails.numberOfRoomsForRent) {
+  formState.propertyDetails.numberOfRoomsForRent = 1
+}
+
+const incrementRooms = () => {
+  formState.propertyDetails.numberOfRoomsForRent++
+}
+
+const decrementRooms = () => {
+  if (formState.propertyDetails.numberOfRoomsForRent > 1) {
```

Comment:
Instead of this condition check, you can disable the Decrease button (so it is not clickable), similarly to what you did for Increase.

Reply:
cleaned that up